deepchem/utils/test/test_equivariance_utils.py

```diff
@@ -0,0 +1,184 @@
+import unittest
+import torch
+from deepchem.utils import equivariance_utils
+
+
+def commutator(A: torch.Tensor, B: torch.Tensor) -> torch.Tensor:
```

Comment:
This is useful and should maybe just be in equivariance_utils.py

Reply:
done.